share/openPMD/thirdParty/pybind11/include/pybind11/pybind11.h

```diff
@@ -10,6 +10,24 @@
 
 #pragma once
 
+/*
```

Comment:
This one seems to fix the CI issues that I saw. Might want to put it in a separate PR and merge it with the MSVC block below.